From f6d14848551d1b3d89ec692c326072dba7dbb32e Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Mon, 13 Aug 2018 10:38:03 -0500
Subject: [PATCH] kernel_validator: add Clear Linux kernel config path

Clear Linux OS store the kernel configuration file in
/usr/lib/kernel. This patch adds this path to the possiblePaths
variable.

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---
 test/e2e_node/system/kernel_validator.go | 1 +
 1 file changed, 1 insertion(+)

diff --git a/test/e2e_node/system/kernel_validator.go b/test/e2e_node/system/kernel_validator.go
index 79681674..916cae2b 100644
--- a/test/e2e_node/system/kernel_validator.go
+++ b/test/e2e_node/system/kernel_validator.go
@@ -177,6 +177,7 @@ func (k *KernelValidator) getKernelConfigReader() (io.Reader, error) {
 	possibePaths := []string{
 		"/proc/config.gz",
 		"/boot/config-" + k.kernelRelease,
+		"/usr/lib/kernel/config-" + k.kernelRelease,
 		"/usr/src/linux-" + k.kernelRelease + "/.config",
 		"/usr/src/linux/.config",
 	}
-- 
2.18.0

